.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
But @IonicaBizau, your tests use the code from dist folder. So if I remove `dist` npm script then future contributors will be confused, they won't be able to test their changes locally. Wouldn't it be better to modify `dist` script to use `babel-it` or whatever you use on your machine ?